install_config/build_defaults_overrides.adoc

```diff
@@ -44,6 +44,47 @@ they are allowed to pull.
 [[setting-global-build-defaults]]
 == Setting Global Build Defaults
 
+You can set global build defaults two ways:
```

Comment:
A nice to have, would be links to: 

- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#using-a-proxy-for-git-cloning
- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#docker-strategy-environment
- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#custom-strategy-environment

While these are _not_ global build settings, I find it important to link the topics of changing build configs, to one another. So users can make informed decisions about when to make a configuration at a global level or on a per-application level. 

The same linking should also occur bi-directionally on the links provided.